// phosphor-react-native/lib/module/bold/CirclesThreePlus.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle, Line } from 'react-native-svg';

function CirclesThreePlus(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 74,
    cy: 74,
    r: 34,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 182,
    cy: 74,
    r: 34,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 74,
    cy: 182,
    r: 34,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 182,
    y1: 156,
    x2: 182,
    y2: 208,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 208,
    y1: 182,
    x2: 156,
    y2: 182,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }));
}

export default CirclesThreePlus;
