// phosphor-react-native/lib/module/bold/MoonStars.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Line, Path } from 'react-native-svg';

function MoonStars(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 216,
    y1: 112,
    x2: 216,
    y2: 64,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 240,
    y1: 88,
    x2: 192,
    y2: 88,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 168,
    y1: 24,
    x2: 168,
    y2: 56,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 184,
    y1: 40,
    x2: 152,
    y2: 40,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M216.7,152.6A91.9,91.9,0,0,1,103.4,39.3h0A92,92,0,1,0,216.7,152.6Z",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }));
}

export default MoonStars;
