// phosphor-react-native/lib/module/bold/Shower.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle, Path } from 'react-native-svg';

function Shower(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 60,
    cy: 228,
    r: 16,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 92,
    cy: 196,
    r: 16,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 28,
    cy: 196,
    r: 16,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 60,
    cy: 164,
    r: 16,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M244,40H219.3a7.9,7.9,0,0,0-5.6,2.3L184,72,64.4,86.1a8,8,0,0,0-4.7,13.6l96.6,96.6a8,8,0,0,0,13.6-4.7L184,72",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 24
  }));
}

export default Shower;
