// phosphor-react-native/lib/module/regular/BugDroid.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path, Line, Circle } from 'react-native-svg';

function BugDroid(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M208,148a80,80,0,0,1-160,0V108a80,80,0,0,1,160,0Z",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 208,
    y1: 128,
    x2: 48,
    y2: 128,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 156,
    cy: 92,
    r: 12,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 100,
    cy: 92,
    r: 12,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 56,
    y1: 28,
    x2: 76.3,
    y2: 46.9,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 200,
    y1: 28,
    x2: 179.7,
    y2: 46.9,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }));
}

export default BugDroid;
