// phosphor-react-native/lib/module/regular/MicrophoneSlash.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path, Line } from 'react-native-svg';

function MicrophoneSlash(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M176.4,181.3A72,72,0,0,1,56.4,136",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M154.9,157.6A39.6,39.6,0,0,1,128,168h0a40,40,0,0,1-40-40V84",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 128,
    y1: 200,
    x2: 128,
    y2: 232,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 48,
    y1: 40,
    x2: 208,
    y2: 216,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M94,43a39.8,39.8,0,0,1,34-19h0a40,40,0,0,1,40,40v60.4",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M199.6,136a72.4,72.4,0,0,1-4.5,18.2",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }));
}

export default MicrophoneSlash;
