// phosphor-react-native/lib/module/regular/SmileyNervous.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle, Polyline } from 'react-native-svg';

function SmileyNervous(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 128,
    r: 96,
    fill: "none",
    stroke: props.color,
    strokeMiterlimit: 10,
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Polyline, {
    points: "80 172 96 152 112 172 128 152 144 172 160 152 176 172",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 92,
    cy: 108,
    r: 12,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 164,
    cy: 108,
    r: 12,
    fill: props.color
  }));
}

export default SmileyNervous;
