// phosphor-react-native/lib/commonjs/duotone/Money.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var React = _interopRequireWildcard(require("react"));

var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function (nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { default: obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj.default = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function Money(props) {
  return /*#__PURE__*/React.createElement(_reactNativeSvg.default, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(_reactNativeSvg.Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Circle, {
    cx: 128,
    cy: 128,
    r: 32,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Circle, {
    cx: 128,
    cy: 128,
    r: 32,
    opacity: 0.2,
    fill: props.color
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Path, {
    d: "M232,64H176l64,56V72A8,8,0,0,0,232,64Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Path, {
    d: "M232,192a8,8,0,0,0,8-8V136l-64,56Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Path, {
    d: "M16,72v48L80,64H24A8,8,0,0,0,16,72Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Path, {
    d: "M16,184a8,8,0,0,0,8,8H80L16,136Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Rect, {
    x: 16,
    y: 64,
    width: 224,
    height: 128,
    rx: 8,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 176,
    y1: 64,
    x2: 240,
    y2: 120,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 176,
    y1: 192,
    x2: 240,
    y2: 136,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 80,
    y1: 64,
    x2: 16,
    y2: 120,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 80,
    y1: 192,
    x2: 16,
    y2: 136,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }));
}

var _default = Money;
exports.default = _default;
