// phosphor-react-native/lib/commonjs/duotone/WindowsLogo.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var React = _interopRequireWildcard(require("react"));

var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function (nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { default: obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj.default = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function WindowsLogo(props) {
  return /*#__PURE__*/React.createElement(_reactNativeSvg.default, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(_reactNativeSvg.Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "216 216 136 201.5 136 201.5 136 144 216 144 216 216",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "104 195.6 40 184 40 144 104 144 104 195.6",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "216 40 136 54.5 136 54.5 136 112 216 112 216 40",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "104 60.4 40 72 40 112 104 112 104 60.4",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "216 216 136 201.5 136 201.5 136 144 216 144 216 216",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "104 195.6 40 184 40 144 104 144 104 195.6",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "216 40 136 54.5 136 54.5 136 112 216 112 216 40",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Polygon, {
    points: "104 60.4 40 72 40 112 104 112 104 60.4",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }));
}

var _default = WindowsLogo;
exports.default = _default;
