// phosphor-react-native/lib/module/fill/ArrowBendRightDown.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path } from 'react-native-svg';

function ArrowBendRightDown(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M205.7,181.7l-48,48a8.2,8.2,0,0,1-11.4,0l-48-48a8.4,8.4,0,0,1-1.7-8.8A8,8,0,0,1,104,168h40V128A88.1,88.1,0,0,0,56,40a8,8,0,0,1,0-16A104.2,104.2,0,0,1,160,128v40h40a8,8,0,0,1,7.4,4.9A8.4,8.4,0,0,1,205.7,181.7Z"
  }));
}

export default ArrowBendRightDown;
