// phosphor-react-native/lib/module/fill/CirclesThreePlus.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle, Path } from 'react-native-svg';

function CirclesThreePlus(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 76,
    cy: 76,
    r: 44,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 180,
    cy: 76,
    r: 44,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 76,
    cy: 180,
    r: 44,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M208,172H188V152a8,8,0,0,0-16,0v20H152a8,8,0,0,0,0,16h20v20a8,8,0,0,0,16,0V188h20a8,8,0,0,0,0-16Z"
  }));
}

export default CirclesThreePlus;
