// phosphor-react-native/lib/module/fill/FadersHorizontal.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path } from 'react-native-svg';

function FadersHorizontal(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M136,164H40a8,8,0,0,0,0,16h96a8,8,0,0,0,0-16Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M216,164H176V148a8,8,0,0,0-16,0v48a8,8,0,0,0,16,0V180h40a8,8,0,0,0,0-16Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M40,92H72a8,8,0,0,0,0-16H40a8,8,0,0,0,0,16Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M104,116a8,8,0,0,0,8-8V92H216a8,8,0,0,0,0-16H112V60a8,8,0,0,0-16,0v48A8,8,0,0,0,104,116Z"
  }));
}

export default FadersHorizontal;
