// phosphor-react-native/lib/module/fill/QrCode.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path } from 'react-native-svg';

function QrCode(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 40,
    y: 40,
    width: 80,
    height: 80,
    rx: 16
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 40,
    y: 136,
    width: 80,
    height: 80,
    rx: 16
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 136,
    y: 40,
    width: 80,
    height: 80,
    rx: 16
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M144,184a8,8,0,0,0,8-8V144a8,8,0,0,0-16,0v32A8,8,0,0,0,144,184Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M208,152H184v-8a8,8,0,0,0-16,0v56H144a8,8,0,0,0,0,16h32a8,8,0,0,0,8-8V168h24a8,8,0,0,0,0-16Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M208,184a8,8,0,0,0-8,8v16a8,8,0,0,0,16,0V192A8,8,0,0,0,208,184Z"
  }));
}

export default QrCode;
