// phosphor-react-native/lib/module/fill/Wall.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path } from 'react-native-svg';

function Wall(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M136,92h92a4,4,0,0,0,4-4V56a8,8,0,0,0-8-8H140a4,4,0,0,0-4,4Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M184,152h44a4,4,0,0,0,4-4V112a4,4,0,0,0-4-4H184Z"
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 88,
    y: 108,
    width: 80,
    height: 44
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M72,108H28a4,4,0,0,0-4,4v36a4,4,0,0,0,4,4H72Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M136,168v36a4,4,0,0,0,4,4h84a8,8,0,0,0,8-8V172a4,4,0,0,0-4-4Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M120,92V52a4,4,0,0,0-4-4H32a8,8,0,0,0-8,8V88a4,4,0,0,0,4,4Z"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M120,168H28a4,4,0,0,0-4,4v28a8,8,0,0,0,8,8h84a4,4,0,0,0,4-4Z"
  }));
}

export default Wall;
