// phosphor-react-native/lib/module/light/DeviceTabletCamera.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle } from 'react-native-svg';

function DeviceTabletCamera(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 32,
    y: 48,
    width: 192,
    height: 160,
    rx: 16,
    transform: "translate(256) rotate(90)",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 68,
    r: 10,
    fill: props.color
  }));
}

export default DeviceTabletCamera;
