// phosphor-react-native/lib/commonjs/light/MathOperations.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var React = _interopRequireWildcard(require("react"));

var _reactNativeSvg = _interopRequireWildcard(require("react-native-svg"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function (nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { default: obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj.default = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function MathOperations(props) {
  return /*#__PURE__*/React.createElement(_reactNativeSvg.default, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(_reactNativeSvg.Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 40,
    y1: 184,
    x2: 104,
    y2: 184,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 72,
    y1: 152,
    x2: 72,
    y2: 216,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 104,
    y1: 72,
    x2: 40,
    y2: 72,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 216,
    y1: 168.1,
    x2: 152,
    y2: 168.1,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 216,
    y1: 199.9,
    x2: 152,
    y2: 199.9,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 208,
    y1: 48,
    x2: 160,
    y2: 96,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }), /*#__PURE__*/React.createElement(_reactNativeSvg.Line, {
    x1: 208,
    y1: 96,
    x2: 160,
    y2: 48,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 12
  }));
}

var _default = MathOperations;
exports.default = _default;
