// phosphor-react-native/lib/module/thin/DiamondsFour.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect } from 'react-native-svg';

function DiamondsFour(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 99.7,
    y: 35.7,
    width: 56.6,
    height: 56.57,
    rx: 8,
    transform: "translate(-7.8 109.3) rotate(-45)",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 8
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 163.7,
    y: 99.7,
    width: 56.6,
    height: 56.57,
    rx: 8,
    transform: "translate(-34.3 173.3) rotate(-45)",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 8
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 35.7,
    y: 99.7,
    width: 56.6,
    height: 56.57,
    rx: 8,
    transform: "translate(-71.8 82.7) rotate(-45)",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 8
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 99.7,
    y: 163.7,
    width: 56.6,
    height: 56.57,
    rx: 8,
    transform: "translate(-98.3 146.7) rotate(-45)",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 8
  }));
}

export default DiamondsFour;
