// phosphor-react-native/lib/module/thin/FilmScript.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle } from 'react-native-svg';

function FilmScript(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 48,
    y: 32,
    width: 160,
    height: 192,
    rx: 8,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 8
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 84,
    cy: 80,
    r: 8,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 84,
    cy: 176,
    r: 8,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 84,
    cy: 128,
    r: 8,
    fill: props.color
  }));
}

export default FilmScript;
