// phosphor-react-native/lib/module/thin/NumberSquareTwo.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path } from 'react-native-svg';

function NumberSquareTwo(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 40,
    y: 40,
    width: 176,
    height: 176,
    rx: 8,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 8
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M105.9,94.7A24,24,0,0,1,152,104a23.6,23.6,0,0,1-4.1,13.4h0L104,176h48",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 8
  }));
}

export default NumberSquareTwo;
