// phosphor-react-native/lib/module/duotone/Bandaids.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path, Circle } from 'react-native-svg';

function Bandaids(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M206.6,94.6a32,32,0,0,0-45.2-45.2L128,82.7,173.3,128Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M49.4,161.4a32,32,0,0,0,45.2,45.2L128,173.3,82.7,128Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M94.6,49.4A32,32,0,0,0,49.4,94.6L82.7,128,128,82.7Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M173.3,128,128,173.3l33.4,33.3a32,32,0,0,0,45.2-45.2Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M49.4,94.6A32,32,0,0,1,94.6,49.4l112,112a32,32,0,0,1-45.2,45.2Z",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M161.4,49.4a32,32,0,0,1,45.2,45.2l-112,112a32,32,0,0,1-45.2-45.2Z",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 128,
    r: 4,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16,
    opacity: 0.5
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 128,
    r: 12,
    fill: props.color
  }));
}

export default Bandaids;
