// phosphor-react-native/lib/module/duotone/DotsThreeVertical.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle } from 'react-native-svg';

function DotsThreeVertical(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 128,
    r: 12,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 64,
    r: 12,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 192,
    r: 12,
    fill: props.color
  }));
}

export default DotsThreeVertical;
