// phosphor-react-native/lib/module/duotone/Jeep.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path, Line, Circle } from 'react-native-svg';

function Jeep(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M220,176v32a8,8,0,0,1-8,8H188a8,8,0,0,1-8-8V176Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M76,176v32a8,8,0,0,1-8,8H44a8,8,0,0,1-8-8V176Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M36,96l7-49.1A8,8,0,0,1,50.9,40H205.1a8,8,0,0,1,7.9,6.9L220,96Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M220,176v32a8,8,0,0,1-8,8H188a8,8,0,0,1-8-8V176",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M76,176v32a8,8,0,0,1-8,8H44a8,8,0,0,1-8-8V176",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 16,
    y1: 96,
    x2: 240,
    y2: 96,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 144,
    y1: 128,
    x2: 144,
    y2: 176,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 112,
    y1: 128,
    x2: 112,
    y2: 176,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M220,176H36V96l7-49.1A8,8,0,0,1,50.9,40H205.1a8,8,0,0,1,7.9,6.9L220,96Z",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 72,
    cy: 132,
    r: 12,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 184,
    cy: 132,
    r: 12,
    fill: props.color
  }));
}

export default Jeep;
