// phosphor-react-native/lib/module/duotone/MediumLogo.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Ellipse, Line } from 'react-native-svg';

function MediumLogo(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Ellipse, {
    cx: 72,
    cy: 128,
    rx: 56,
    ry: 60,
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Ellipse, {
    cx: 184,
    cy: 128,
    rx: 24,
    ry: 56,
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Ellipse, {
    cx: 72,
    cy: 128,
    rx: 56,
    ry: 60,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Ellipse, {
    cx: 184,
    cy: 128,
    rx: 24,
    ry: 56,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 240,
    y1: 72,
    x2: 240,
    y2: 184,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }));
}

export default MediumLogo;
