// phosphor-react-native/lib/module/duotone/Money.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Circle, Path, Line } from 'react-native-svg';

function Money(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 128,
    r: 32,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 128,
    r: 32,
    opacity: 0.2,
    fill: props.color
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M232,64H176l64,56V72A8,8,0,0,0,232,64Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M232,192a8,8,0,0,0,8-8V136l-64,56Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M16,72v48L80,64H24A8,8,0,0,0,16,72Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M16,184a8,8,0,0,0,8,8H80L16,136Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Rect, {
    x: 16,
    y: 64,
    width: 224,
    height: 128,
    rx: 8,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 176,
    y1: 64,
    x2: 240,
    y2: 120,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 176,
    y1: 192,
    x2: 240,
    y2: 136,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 80,
    y1: 64,
    x2: 16,
    y2: 120,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 80,
    y1: 192,
    x2: 16,
    y2: 136,
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }));
}

export default Money;
