// phosphor-react-native/lib/module/duotone/SkipForwardCircle.js
function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

/* GENERATED FILE */
import * as React from 'react';
import Svg, { Rect, Path, Circle, Polygon, Line } from 'react-native-svg';

function SkipForwardCircle(props) {
  return /*#__PURE__*/React.createElement(Svg, _extends({
    viewBox: "0 0 256 256",
    width: props.size,
    height: props.size,
    fill: props.color
  }, props), /*#__PURE__*/React.createElement(Rect, {
    width: 256,
    height: 256,
    fill: "none"
  }), /*#__PURE__*/React.createElement(Path, {
    d: "M128,32a96,96,0,1,0,96,96A96,96,0,0,0,128,32ZM100,160V96l48,32Z",
    opacity: 0.2
  }), /*#__PURE__*/React.createElement(Circle, {
    cx: 128,
    cy: 128,
    r: 96,
    fill: "none",
    stroke: props.color,
    strokeMiterlimit: 10,
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Polygon, {
    points: "148 128 100 96 100 160 148 128",
    fill: "none",
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }), /*#__PURE__*/React.createElement(Line, {
    x1: 156,
    y1: 96,
    x2: 156,
    y2: 160,
    stroke: props.color,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeWidth: 16
  }));
}

export default SkipForwardCircle;
